### DISCUSSION TITLE
Do NOT delete the virtual machine!

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2643] [Show and tell]
If you get a notification from Nightscout on Google Cloud telling you that you should delete the virtual machine, please don't.

I have a PR waiting for review.  It should be reviewed today (Jan. 23).  Then, it should be merged shortly after. 

If you have got the notification, after having attempted to update the platform, please wait for the merge.  I will post here informing everyone that the new update is available.  At that point, if opening a terminal using SSH does not automatically open a menu, just run the bootstrap (https://navid200.github.io/xDrip/docs/Nightscout/NS_SyncExecutables.html) and everything will go back to normal.

**Comment by Navid200** [comment]
The problem is resolved.
You can update following these instructions as before:  https://navid200.github.io/xDrip/docs/Nightscout/NS_SyncExecutables.html

**Comment by marionbarker** [comment]
Thank you. I lost contact with my Nightscout site (no menu on console) in preparation for updating to dev, but Nightscout kept working. I have 3 sites - one for real and two for test. The first test site update went fine. The real site did not.

I've been too busy to reach out earlier - so I am very happy to find this discussion.

In the meantime, I cleared out a test site so I could copy all my data from my real site to my test site. Once that is done and I have a backup of my full mongo db, I will attempt the new bootstrap.

Lesson learned - backup mongo db before doing an update.

**Reply by Navid200** [reply]
If you type the following in the terminal, does the menu come up?
/xDrip/scripts/menu.sh

**Comment by marionbarker** [comment]
Giving you the play-by-play in case it is helpful.

Initial console message:
```
Last login: Sat Jan 21 16:57:56 2023 from 35.235.244.0
-bash: /xDrip/scripts/menu.sh: No such file or directory
```
I ran this command:
```
curl https://raw.githubusercontent.com/jamorham/nightscout-vps/vps-1/bootstrap.sh | bash
```

That did some stuff, then paused then I got this menu (which I saw last time but OK did not help). 
This time OK worked:
![Screenshot 2023-01-26 at 4 05 58 PM](https://user-images.githubusercontent.com/19607791/214977897-c5d62abc-1700-4d19-a4b7-810f8f5bbd02.png)

I hit OK.
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'python-is-python2' instead of 'python'
g++ is already the newest version (4:9.3.0-1ubuntu2).
gcc is already the newest version (4:9.3.0-1ubuntu2).
make is already the newest version (4.2.1-1.2).
python-is-python2 is already the newest version (2.7.17-4).
git is already the newest version (1:2.25.1-1ubuntu3.8).
0 upgraded, 0 newly installed, 0 to remove and 26 not upgraded.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
netcat is already the newest version (1.206-1ubuntu1).
0 upgraded, 0 newly installed, 0 to remove and 26 not upgraded.
sudo: ConfigServer: command not found
```

Waited again and I get this (I think that message needs to be updated):

![Screenshot 2023-01-26 at 4 10 30 PM](https://user-images.githubusercontent.com/19607791/214978336-07cd16fa-d588-4c0a-ab87-8d1cf5ff4fbe.png)

I hit ok - it says next steps will take about 30 sec.

And I see missing packages. So I will run the Update Platform from the Google Submenu again.
Wish me luck.

**Reply by marionbarker** [reply]
That worked.  My status has no red font.

**Reply by Navid200** [reply]
Great.
